src/ar/read_landmarks.cpp

```diff
@@ -76,14 +76,24 @@ void detectLandmarksLoop(){
 			}
 
 			landmark_lock.lock();
-			current_landmarks = output;
+			for (int i = 0; i < NUM_LANDMARKS; i++) {
+				if (output[i](2) != 0.0) current_landmarks[i] = output[i];
+			}
```

Comment:
Just to make sure I understand this correctly, does this make the landmark detection "sticky," i.e. store all landmarks seen during the time period between calls to `readLandmarks()`, rather than replacing them every frame?